README.md

```diff
@@ -1,10 +1,15 @@
 # linked-list
 Implements a linked list with python
 
-This repo holds sample code for some classic data structures, implemented in Python, including singly- and doubly-linked lists and stacks. Singly-linked lists are convenient for keeping a group of data nodes in order; adding and removing them is simple and efficient. They are most useful when we only need to traverse our data in one direction. Doubly-linked lists make their nodes more accessible since they can be inserted or removed at either end of the list; but navigating the list is costlier.
+This repo holds sample code for some classic data structures, implemented in Python, including singly- and doubly-linked lists and stacks, queues, and deques. Singly-linked lists are convenient for keeping a group of data nodes in order; adding and removing them is simple and efficient. They are most useful when we only need to traverse our data in one direction. Doubly-linked lists make their nodes more accessible since they can be inserted or removed at either end of the list; but navigating the list is costlier. Stacks, queues, and deques are more refined implementations.
```

Comment:
As your data structures repo gets larger, you might want to consider breaking this out into separate paragraphs and sections that describe each data structure. This paragraph will get pretty dense before long (plus employers like that).